### PR TITLE
Remove redundant entries

### PR DESCRIPTION
**Pull request by iam-py-test**
This should remove a few redundant entries

### CODE REVIEW

File: filters-share/adservice.txt [hunk 1]
```diff
@@ -115,7 +115,6 @@
 ||ad.adjw.co.kr^
 ||compass.adop.cc^
 ||catad5959.com^
-||adex.ednplus.com^
```
Comment (piquark6046) [inline filters-share/adservice.txt:118]: This is used on https://www.chosun.com/politics/politics_general/2021/09/20/KBX3Y25ZRJF55CRPEKWEWXJNCY/.

File: filters-share/adservice_firstparty.txt [hunk 2]
```diff
@@ -34,7 +34,6 @@
 ||ad-img.ebaykorea.com^
 ||api.ezadsystem.com^
 ||ad.planbplus.co.kr^
-||api.linkmine.co.kr^
```
Comment (piquark6046) [inline filters-share/adservice_firstparty.txt:37]: [Its website](http://linkmine.co.kr/) exists.

File: filters-share/specific_URL.txt [hunk 3]
```diff
@@ -354,7 +354,6 @@
 ||hellodd.com/adManager/css/adManager.css
 ||hellodd.com/bannerpop/uploads/
 ||hellot.net/data/banner/
-||dailysecu.com/bannerpop/uploads/
```
Comment (piquark6046) [inline filters-share/specific_URL.txt:357]: This blocks advertisement on https://dailysecu.com/.

File: filters-share/specific_URL.txt [hunk 4]
```diff
@@ -431,7 +430,6 @@
 ||yeongnam.com/web/js/statAd.js
 ||yeongnam.com/mnt/webdata/ad/
 ||5donews.co.kr/layouts/5donews/images/banner/
-||womennews.co.kr/adManager/css/adManager.css
```
Comment (piquark6046) [inline filters-share/specific_URL.txt:434]: This blocks advertisement on http://www.womennews.co.kr/.

**Review by piquark6046 — CHANGES_REQUESTED** [review]

File: filters-share/adservice.txt (hunk 1 above)
Comment (iam-py-test) [reply filters-share/adservice.txt:118]: I know. It still exists on https://github.com/iam-py-test/List-KR/blob/master/filters-share/adservice.txt#L90 (that entry is there twice)

File: filters-share/adservice_firstparty.txt (hunk 2 above)
Comment (iam-py-test) [reply filters-share/adservice_firstparty.txt:37]: The entry still exists at https://github.com/iam-py-test/List-KR/blob/master/filters-share/adservice_firstparty.txt#L31 (I should have clarified that in my original PR, sorry)

File: filters-share/specific_URL.txt (hunk 3 above)
Comment (iam-py-test) [reply filters-share/specific_URL.txt:357]: This entry appears twice: https://github.com/List-KR/List-KR/blob/master/filters-share/specific_URL.txt#L235 and https://github.com/List-KR/List-KR/blob/master/filters-share/specific_URL.txt#L358

File: filters-share/specific_URL.txt (hunk 4 above)
Comment (iam-py-test) [reply filters-share/specific_URL.txt:434]: This entry exists at https://github.com/List-KR/List-KR/blob/master/filters-share/specific_URL.txt#L435 and https://github.com/List-KR/List-KR/blob/master/filters-share/specific_URL.txt#L439

**Review by piquark6046 — APPROVED** [review]

**Comment by iam-py-test** [comment]
Thanks

**Pull request merged**